recipes/txtorcon/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "txtorcon" %}
+{% set version = "0.19.3" %}
+{% set sha256 = "f73396667909a3c7a98f4dd865edf4ed6a2518ee5a935d92e18b8a479ec244fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - twisted >=15.5.0
+    - ipaddress >=1.0.16  # [py2k]
+    - zope.interface >=3.6.1
+    - incremental
+    - automat
+
+test:
+  requires:
+    - pyopenssl
```

Comment:
I'm not sure which pieces of twisted require which packages in the "tls extras" and what is the overlap with what `txtorcon` actually uses from twisted that might be picked up at runtime. If it was feasible to run the tests maybe do that? Otherwise I think it might be ok to just include `pyopenssl` in the absence of further evidence of needing the other packages.